pkg/kvm/transport.go

```diff
@@ -38,7 +38,7 @@ func NewNvmeVfiouserTransport(ctrlrDir string) frontend.NvmeTransport {
 }
 
 func (c *nvmeVfiouserTransport) Params(ctrlr *pb.NvmeController, nqn string) (spdk.NvmfSubsystemAddListenerParams, error) {
-	pcieID := ctrlr.Spec.PcieId
```

Comment:
I am thinking about moving vfiouser transport into frontend in a following PR, so we could test pcie transport with godpu... @glimchb wdyt?